zhusuan/distributions/multivariate.py

```diff
@@ -403,3 +372,254 @@ def _log_prob(self, given):
 
     def _prob(self, given):
         return tf.exp(self._log_prob(given))
+
+
+class ExpConcrete(Distribution):
```

Comment:
I think the relationship between them is more like a kind of transform, and if use inheritance we mean ExpConcrete *is* Concrete, but it is actually not.